### PR TITLE
[WIP][bsp] add eastsoft es32f0654 bsp

### PR DESCRIPTION
**Pull request by wangyq2018**
## 拉取/合并请求描述：(PR description)

[ [bsp] add eastsoft es32f0654 bsp ]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [x] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [ ] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other style
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality

### CODE REVIEW

File: bsp/eastsoft/es32f0654/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# ES-PDS-ES32F0654 开发板 BSP 说明
+标签： EastSoft、国产MCU、Cortex-M0、ES32F0654LT
+
+## 1. 简介
+
+本文档为上海东软载波微电子开发团队为 ES-PDS-ES32F0654 开发板提供的 BSP (板级支持包) 说明。
+通过阅读本文档，开发者可以快速地上手该 BSP，将 RT-Thread 运行在开发板上。
+
```
Comment (BernardXiong) [inline bsp/eastsoft/es32f0654/README.md]: 建议提供一份开发板的照片

File: bsp/eastsoft/es32f0654/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+更多详细信息请咨询[上海东软载波微电子技术支持](http://www.essemi.com/)
+
+## 2. 快速上手
+
+本 BSP 为开发者提供 MDK5 工程。下面以 MDK5 开发环境为例，介绍如何将系统运行起来。
+
+### 硬件连接
+
+使用 ESLinkⅡ (自带 CDC 串口)或 Jlink 等调试工具连接开发板到 PC，拨动开关选择使用调试工具供电或使用外部电源供电。若使用 Jlink 等调试工具，还需要将 UART2_TX(PC12)、UART2_RX(PD2)、GND 接到串口工具上。
+
```
Comment (BernardXiong) [inline bsp/eastsoft/es32f0654/README.md]: 如果有连接相关图片更佳

**Comment by wangyq2018** [comment]
@BernardXiong 因为考虑到后续会再添加不同的芯片，所以单独建立了一个以公司命名的文件夹。或者还按照原先的以芯片的方式建立文件夹？

**Comment by BernardXiong** [comment]
不建议按照公司命名啊，还是多以芯片系列来命名吧，公司命名感觉有些怪异

**Pull request closed without merging**